x-pack/plugins/security_solution/public/detections/components/rules/step_rule_actions/actions_view.test.tsx

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { render } from '@testing-library/react';
+import { FrequencyDescription } from './notification_action';
+import { TestProviders } from '../../../../common/mock';
+
+describe('getFrequencyDescription', () => {
+  it('should return empty string if frequency is not specified', () => {
+    const { container } = render(<FrequencyDescription />, {
+      wrapper: TestProviders,
+    });
+
+    expect(container).toBeEmptyDOMElement();
+  });
+
+  it('should correctly handle "For each alert. Per rule run."', async () => {
+    const frequency = { notifyWhen: 'onActiveAlert', summary: false, throttle: null } as const;
+
+    const { findByText } = render(<FrequencyDescription frequency={frequency} />, {
+      wrapper: TestProviders,
+    });
+
+    expect(await findByText('For each alert. Per rule run.')).toBeInTheDocument();
+  });
+
+  it('should correctly handle "Summary of alerts. Per rule run."', async () => {
+    const frequency = { notifyWhen: 'onActiveAlert', summary: true, throttle: null } as const;
+
+    const { findByText } = render(<FrequencyDescription frequency={frequency} />, {
+      wrapper: TestProviders,
+    });
+
+    expect(await findByText('Summary of alerts. Per rule run.')).toBeInTheDocument();
+  });
+
+  it('should return empty string if type is "onThrottleInterval" but throttle is not specified', () => {
```

Comment:
This should not be possible. Have you seen this issue?

Reply:
Actually, no. Just decided to make sure it won't cause an issue if we forget to pass it somehow.